src/Modal.js

```diff
@@ -120,6 +122,7 @@ class Modal extends React.Component {
         return { ...styles.overlay }
       }
       case 'body': {
+        if (className) return { ...styles.bodyWithClassName, maxWidth }
```

Comment:
Thank a lot to PR,

There is one thing to change. The  condition to check `className`  now will return  `styles.bodyWithClassName` always  cause default props `className` is not **false**, **undefined** , **null** or **0**